docs/concepts-broker-load-balancing-concepts.md

```diff
@@ -492,40 +492,41 @@ Unload all bundles in a namespace
-TransferShedder|Unload bundles from the **highest** load brokers to the **lowest** load brokers until the standard deviation of the broker load distribution is below the configured threshold.|This is the **default** strategy for the **extensible** load balancer. <br/><br/>It pre-assigns destination brokers when unloading.|Pulsar 3.0 and later versions.<br/><br/>This strategy is **only available** in the **extensible** load balancer.
+Bundle unloading strategy|Definition| When to use                                                                                                                         |Available version
+|---|---|-------------------------------------------------------------------------------------------------------------------------------------|---
+OverloadShedder|Unload bundles on brokers if a **broker's maximum resource usage** exceeds the configured threshold.| Use when you want to set broker usage below a threshold.                                                                            |Pulsar 1.18 and later versions.<br/><br/> This strategy is **only available** in the **modular** load balancer.
+ThresholdShedder|Unload bundles if a broker's average usage is greater than the** cluster average usage** plus **configured threshold**. | Use when you want to evenly spread loads across all brokers base on cluster average usage.                                          | Pulsar 2.6 and later versions.<br/><br/> This strategy is **only available** in the **modular** load balancer.
+UniformLoadShedder|Distribute load uniformly across all brokers, based on **minimal** and **maximum** load.| Use when you want to compare the minimal and maximum loaded brokers.                                                                |Pulsar 2.10.0 and later versions.<br/><br/> This strategy is **only available** in the **modular** load balancer.
+TransferShedder|Unload bundles from the **highest** load brokers to the **lowest** load brokers until the standard deviation of the broker load distribution is below the configured threshold.| This is the **default** strategy for the **extensible** load balancer. <br/><br/>It pre-assigns destination brokers when unloading. |Pulsar 3.0 and later versions.<br/><br/>This strategy is **only available** in the **extensible** load balancer.
+AvgShedder|Unload bundles to keep the range of broker resource usage within the configured threshold.| Use when you want to achieve greate service stability and load switching accuracy.                                                  |Pulsar 3.0 and later versions. <br/><br/>If you run version greater than 2.9 but below 3.0, you can try to cherry pick into your repository easily: https://github.com/apache/pulsar/pull/22946<br/><br/>This strategy is only available in the modular load balancer.
```

Comment:
Can you describe which specific version supports the `AvgShedder`.

Reply:
Updated.